packages/dds/tree/src/test/shared-tree-core/edit-manager/editManager.spec.ts

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { runBenchmark } from "./editManager.bench";
+import { testCodec } from "./editManagerCodecs.spec";
+import { testCorrectness } from "./editManagerCorrectness.spec";
+import { testPerf } from "./editManagerPerf.spec";
+
+describe("EditManager", () => {
+	testCodec();
+	testCorrectness();
+	testPerf();
+	runBenchmark();
```

Comment:
do the bench scripts work correctly with this not in a .bench.ts file?

Reply:
Good question... It looks like it works fine but it's entirely possibly some scrips would break down so I'll refrain from moving that one.